libvcx/src/connection.rs

```diff
@@ -638,16 +752,19 @@ pub mod tests {
     #[test]
     fn test_send_and_download_messages() {
         let _setup = SetupLibraryAgencyV2::init();
-        let (alice_to_faber, faber_to_alice) = connection::tests::create_connected_connections(None, None);
+        let mut institution = Faber::setup();
+        let mut consumer1 = Alice::setup();
```

Comment:
Why `consumer1`? We have only one consumer in this test

Reply:
updated